.github/ISSUE_TEMPLATE/cherry_pick_template.md

```diff
@@ -0,0 +1,26 @@
+**Replace *everything* in angle brackets in the title AND body of this issue.  If you have any questions see the [cherry pick documentation](https://github.com/ampproject/amphtml/blob/master/contributing/release-schedule.md#cherry-picks).**
+
+*GitHub issue your cherry pick is fixing:*
+
+Issue #<ISSUE_NUMBER>
+
+*PR that you are requesting a cherry pick for:*
+*(Put N/A if you do not yet have a PR with a fix if you are seeking pre-approval for a cherry pick.  Edit this issue's body to add the PR number once it is available.)*
+
+PR #<PR_NUMBER>
```

Comment:
should we also add the SHA ? since the SHA is what you need to accomplish the cherry pick. (it will be in the PR but slightly hard to find)

Reply:
If it's straightforward enough for the onduty person to map from PR#->SHA then I think I'd prefer to just leave that as a step for the onduty person to handle (since most people just think in terms of PRs as far as I can tell).

Is there a scenario where just using the PR # would be ambiguous and the SHA would resolve that ambiguity?

Reply:
To get this in before the new release process goes in, I'll follow up with you offline about this, and I'll send another PR in the future if we decide to add this.

Reply:
depending on the merge mode for example, a PR can have multiple SHAs merged into master. currently we squash meaning all the multiple SHAs in a PR get turned into 1 SHA all the time

Reply:
Got it; would it make sense for the onduty person to sort this out (with some instructions documented for them)?  I'm worried that the person requesting the cherry pick who isn't familiar with SHAs might choose the wrong one so we'd want onduty double checking it anyway.

Reply:
i agree. i think lets just keep it as is for now. We don't really allow for non squash merges anymore (I can't remember if it is enforced through github or not)